R/QueueDataWarehouse.R

```diff
@@ -71,7 +72,7 @@
 QueueDataWarehouse <- function(reportsuite.id, date.from, date.to, metrics, elements,
                                date.granularity='day', segment.id='', data.current=TRUE,
                                expedite=FALSE, interval.seconds=5, max.attempts=120,
-                               validate=TRUE,enqueueOnly=TRUE, ftp=ftp) {
+                               validate=TRUE,enqueueOnly=TRUE, ftp=ftp, format='csv') {
```

Comment:
I could probably be convinced, but on first glance, I'm not sure that `format` should be a top-level argument in public functions. We're not actually offering the ability to return a csv, it's an implementation detail for the branch where `enqueueOnly` is false. Meaning, you don't get to choose what format is available for `enqueueOnly = TRUE`, the only choice is to write a CSV to your FTP location.